interface/src/index.d.ts

```diff
@@ -799,6 +799,16 @@ declare class ZoweZLUXResources {
   static globalization: ZLUX.Globalization;
 }
 
+interface Window {
+  GIZA_PLUGIN_TO_BE_LOADED?: string;
+  GIZA_SIMPLE_CONTAINER_REQUESTED?: boolean;
+  GIZA_ENVIRONMENT?: string;
+  ZOWE_SWM_SHOW_LOGIN?: number;
+  ZoweZLUX: typeof ZoweZLUXResources | DSMResources;
```

Comment:
The clause `| DSMResources` is not needed here.

Reply:
It hides some mistakes in zlux-app-manager's code, please see my comments on https://github.com/zowe/zlux-app-manager/pull/292

Reply:
My conclusion: I am not adding COM_RS_COMMON_LOGGER into window nor ZoweZLUX into window with my code changes, they already exist (because that's the way it has to work for bootstrap unless you see another solution?) But I am typing them in platform instead of where they are typed in those files.
So window.ZoweZLUX has to exist, but ZoweZLUX is optional & there for convenience